File: Selenium/Activities/Activity22.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;


public class Activity22 {
    public static void main(String[] args) {
        WebDriver driver = new FirefoxDriver();
        Actions builder = new Actions(driver);

        driver.get("https://v1.training-support.net/selenium/popups");
        System.out.println(driver.getTitle());

        WebElement Sign = driver.findElement(By.className("orange"));

        builder.moveToElement(Sign).build().perform();

        System.out.println(Sign.getAttribute("data-tooltip"));

        Sign.click();

        driver.findElement(By.xpath("//*[@id=\"username\"]")).sendKeys("admin");
        driver.findElement(By.xpath("//*[@id=\"password\"]")).sendKeys("password");

        driver.findElement(By.xpath("/html/body/div[4]/div/div[2]/div/div/button")).click();

        WebElement msg = driver.findElement(By.id("action-confirmation"));
        System.out.println(msg.getText());

        driver.close();
    }
}

File: Selenium/Activities/Activity22.py
from selenium import webdriver
from selenium.webdriver.common.by import By


driver = webdriver.Firefox()
actions = webdriver.ActionChains(driver)

driver.get("https://v1.training-support.net/selenium/popups")
print(driver.title)

x = driver.find_element(By.CLASS_NAME, "orange")
# Move the cursor over the button to show tooltip
actions.move_to_element(x).perform()

print(x.get_attribute("data-tooltip"))

x.click()

username = driver.find_element(By.XPATH, '//*[@id="username"]')
password = driver.find_element(By.XPATH, '//*[@id="password"]')
    # Enter credentials
username.send_keys("admin")
password.send_keys("password")
    # Click login
driver.find_element(By.XPATH, "/html/body/div[4]/div/div[2]/div/div/button").click()

message = driver.find_element(By.ID, "action-confirmation")
print("Confirmation message: ", message.text)

driver.close